commit 2aad202fcd4cc9134f1b68b66220f077c7732261
Author: Arnd Bergmann <arnd@arndb.de>
Date:   Tue Aug 14 00:37:18 2018 +0200

    fsi: master-ast-cf: select GENERIC_ALLOCATOR
    
    [ Upstream commit 64999fa7aa2c076ec6d05aee481f11f5296ceb8c ]
    
    In randconfig builds without CONFIG_GENERIC_ALLOCATOR, this driver
    fails to link:
    
    ERROR: "gen_pool_alloc_algo" [drivers/fsi/fsi-master-ast-cf.ko] undefined!
    ERROR: "gen_pool_fixed_alloc" [drivers/fsi/fsi-master-ast-cf.ko] undefined!
    ERROR: "of_gen_pool_get" [drivers/fsi/fsi-master-ast-cf.ko] undefined!
    ERROR: "gen_pool_free" [drivers/fsi/fsi-master-ast-cf.ko] undefined!
    
    Select the dependency as all other users do.
    
    Fixes: 6a794a27daca ("fsi: master-ast-cf: Add new FSI master using Aspeed ColdFire")
    Signed-off-by: Arnd Bergmann <arnd@arndb.de>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/drivers/fsi/Kconfig b/drivers/fsi/Kconfig
index af3a20dd5aa4..99c99a5d57fe 100644
--- a/drivers/fsi/Kconfig
+++ b/drivers/fsi/Kconfig
@@ -46,6 +46,7 @@ config FSI_MASTER_AST_CF
 	tristate "FSI master based on Aspeed ColdFire coprocessor"
 	depends on GPIOLIB
 	depends on GPIO_ASPEED
+	select GENERIC_ALLOCATOR
 	---help---
 	This option enables a FSI master using the AST2400 and AST2500 GPIO
 	lines driven by the internal ColdFire coprocessor. This requires
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1082_2aad202fcd4cc9134f1b68b66220f077c7732261.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1082_2aad202fcd4cc9134f1b68b66220f077c7732261.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
